crates/samples/direct3d12/src/main.rs

```diff
@@ -426,17 +426,17 @@ mod d3d12_hello_triangle {
         let asset_path = exe_path.parent().unwrap();
         let shaders_hlsl_path = asset_path.join("shaders.hlsl");
         let shaders_hlsl = shaders_hlsl_path.to_str().unwrap();
-        let shaders_hlsl = PCWSTR::from(&shaders_hlsl.into());
+        let shaders_hlsl: HSTRING = shaders_hlsl.into();
```

Comment:
A temporary `HSTRING` was being created and then used in the subsequent calls to `D3DCompileFromFile`.